cmd/containerd-shim-runhcs-v1/stats/stats.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package containerd.runhcs.stats.v1;
+
+import weak "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "github.com/containerd/cgroups/metrics.proto";
+
+option go_package = "github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1/stats;stats";
+
+message Statistics {
+	oneof container {
+		WindowsContainerStatistics windows = 1;
+		io.containerd.cgroups.v1.Metrics linux = 2;
+	}
+	VirtualMachineStatistics vm = 3;
+}
+
+message WindowsContainerStatistics {
+	google.protobuf.Timestamp timestamp = 1 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+	google.protobuf.Timestamp container_start_time = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+	int32 uptime_100_ns = 3;
+	WindowsContainerProcessorStatistics processor = 4;
+	WindowsContainerMemoryStatistics memory = 5;
+	WindowsContainerStorageStatistics storage = 6;
+}
+
+message WindowsContainerProcessorStatistics {
+	int32 total_runtime_100_ns = 1;
+	int32 runtime_user_100_ns = 2;
+	int32 runtime_kernel_100_ns = 3;
+}
+
+message WindowsContainerMemoryStatistics {
+	int32 memory_usage_commit_bytes = 1;
+	int32 memory_usage_commit_peak_bytes = 2;
+	int32 memory_usage_private_working_set_bytes = 3;
+}
+
+message WindowsContainerStorageStatistics {
+	int32 read_count_normalized = 1;
+	int32 read_size_bytes = 2;
+	int32 write_count_normalized = 3;
+	int32 write_size_bytes = 4;
+}
+
+message VirtualMachineStatistics {
+	VirtualMachineProcessorStatistics processor = 1;
+	VirtualMachineMemoryStatistics memory = 2;
+}
+
+message VirtualMachineProcessorStatistics {
+	uint64 total_runtime_ns = 1;
+}
+
+message VirtualMachineMemoryStatistics {
+	uint64 working_set_bytes = 1;
```

Comment:
@kevpar - Made this so it should merge with your changes without breaking anything just fine.